Tests/RLPTests.cs

```diff
@@ -0,0 +1,190 @@
+using System;
+using System.Linq;
+using Nethereum.Hex.HexConvertors.Extensions;
+using NUnit.Framework;
+using Web3Unity.Scripts.Library.Ethers.RLP;
+
+namespace Tests
+{
+    public class RLPTests
+    {
+        [Test]
+        [TestCase("0xff", "0x81ff", TestName = "encode_singleHigh")]
```

Comment:
We can probably move all these input into a json file and load that file, but we can leave it like this for now